test/functional/Java8andUp/playlist.xml

```diff
@@ -655,6 +655,28 @@
 		 </impls>
 	</test>
 
+	<test>
+		<testCaseName>TestJstat</testCaseName>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_JDK_HOME)$(D)lib$(D)tools.jar$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
+	-Dcom.ibm.tools.attach.enable=yes \
+	-Djdk.attach.allowAttachSelf=true \
```

Comment:
What is this for? `jstat` doesn't appear to allow attaching to itself in any case.

Reply:
Technically this `-Djdk.attach.allowAttachSelf=true` is passed to the JVM running the test which launches `jstat` however I don't see the requirement for this `JVM` attaching to itself hence remove it.
There are same settings for other tests, leaving them as-is, will leave to a separated PR if needed.

Reply:
I believe that property is relevant to the other tests.

Reply:
Those tests for `TestJps`, `TestJstack`, `TestJmap` and `TestJcmd` are launched similarly like `Jstat` in this PR, and `-Djdk.attach.allowAttachSelf=true` is not needed. Again, I would like leave such refactoring to a separated PR instead.

Reply:
It's not refactoring: you've added a new test - that addition should not include the unnecessary option.

Reply:
I see you've removed it. Thanks.

Reply:
The refactoring referred `-Djdk.attach.allowAttachSelf=true` for other existing tests such as `TestJps`, `TestJstack`, `TestJmap` and `TestJcmd`.

`TestJstat` has removed the option in question.

Reply:
I didn't say anything about the other tests.

Reply:
Alright.